packages/react-composites/tests/browser/meeting/MeetingComposite.test.ts

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { dataUiId } from '../common/utils';
+import { test } from './fixture';
+import { expect, Page } from '@playwright/test';
+
+export const waitForMeetingCompositeToLoad = async (page: Page): Promise<void> => {
+  await page.waitForLoadState('load');
+
+  // @TODO
+  // We wait 3 sec here to work around flakiness due to timing.
```

Comment:
I don't think we need a timeout for call composite to load. It's for chat due to this bug
![image](https://user-images.githubusercontent.com/9782747/134553175-c66ecdfb-4022-4149-8814-cc10cd673600.png)

Could we update the comments to reflect the proper reason?
If that's not the case, please do correct me.

Reply:
Sigh, divergence from main again, this isn't need anymore since the fix to wait for video ready, I will remove

Reply:
Does this fix the chat bug as well?

Reply:
This just loads the configuration screen (chat hasn't loaded at this point). But haven't seen any issues/failures yet with test for checking the chat messaging inside the composite